server/service/service_campaigns.go

```diff
@@ -162,8 +162,12 @@ func (svc Service) StreamCampaignResults(ctx context.Context, conn *websocket.Co
 		// any results are written, to avoid the frontend showing "x of
 		// 0 Hosts Returning y Records")
 		select {
-		case res := <-readChan:
+		case res, ok := <-readChan:
```

Comment:
small nitpick, this switch case only has one case in it. Can we change this for loop to 

```
for res := range reachChan {
...
}
_ = svc.logger.Log("msg", "Redis closed the channel on us!", "err", nil)
return
```
this should work the same

Reply:
alternatively, move the following to the top level switch case
```
case <-ticker.C:
```

Reply:
> this switch case only has one case in it

I think it has 2, the ticker is in this same switch.

Reply:
Yes, it has two, I can definitely move the ticker's case up to make the code more clear.

Reply:
Sorry, I got confused by the way indentation was being displayed in github. Disregard.